src/components/SearchForm/SearchForm.tsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react'
+
+export interface SearchFormProps {
+    title: string,
+    placeHolder?: string
+}
+
+const SearchForm: React.FC<SearchFormProps> = ({ title, placeHolder }): JSX.Element => {
+    return (
+        <form className="c-searchform" role="search" method="get" action="https://carleton.ca">
```

Comment:
Please add an action optional action props 
Please add an onSubmit  props

Reply:
fixed added optional action prop and onSubmit props